src/libcadet/model/reaction/MichaelisMentenReaction.cpp

```diff
@@ -28,313 +28,584 @@
 
 /*<codegen>
 {
-	"name": "MichaelisMentenParamHandler",
-	"externalName": "ExtMichaelisMentenParamHandler",
-	"parameters":
-		[
-			{ "type": "ScalarReactionDependentParameter", "varName": "vMax", "confName": "MM_VMAX"},
-			{ "type": "ScalarReactionDependentParameter", "varName": "kMM", "confName": "MM_KMM"},
-			{ "type": "ComponentDependentReactionDependentParameter", "varName": "kInhibit", "confName": "MM_KI"}
-		]
+    "name": "MichaelisMentenParamHandler",
+    "externalName": "ExtMichaelisMentenParamHandler",
+    "parameters":
+        [
+            { "type": "ScalarReactionDependentParameter", "varName": "vMax", "confName": "MM_VMAX"},
+            { "type": "ComponentDependentReactionDependentParameter", "varName": "kMM", "confName": "MM_KMM"},
+            
+            { "type": "ComponentDependentReactionDependentParameter", "varName": "kInhibitComp", "confName": "MM_KI_C"},
+            { "type": "ComponentDependentReactionDependentParameter", "varName": "kInhibitUnComp", "confName": "MM_KI_UC"},
+            { "type": "ComponentDependentReactionDependentParameter", "varName": "kInhibit", "confName": "MM_KI"}
+        ]
 }
 </codegen>*/
 
 /* Parameter description
  ------------------------
+ MM_VMAX    - Maximum reaction rate (dim: number of reactions)
+ MM_KMM     - Michaelis-Menten constant for each component (dim: number of reactions x number of components)
+ MM_KI_C    - Competitive inhibition constants (K̃_{i,j,k}) ( dim: number of reactions x number of components x number of componetns)
+ MM_KI_UC   - Uncompetitive inhibition constants (K_{i,j,k}) (dim: number of reactions x number of components x number of componetns)
+ MM_KI      - Old infterface for non competative inhibition constants (K_{i,k}) (dim: number of reactions x number of componetns) 
```

Comment:
```suggestion
 MM_KI      - Old infterface for non competative inhibition constants (K_{i,k}) (dim: number of reactions x number of componetns) 
```
 MM_KI      - Old interface for non competitive inhibition constants (K_{i,k}) (dim: number of reactions x number of components)